test/integration/01-setup/01-connection.spec.ts

```diff
@@ -21,6 +27,22 @@ it("connects to bitcoind", async () => {
   expect(chain).toEqual("regtest")
 })
 
+it("detect active bitcoind zeromq notifications", async () => {
+  const notifications = await bitcoindClient.getZmqNotifications()
+  expect(notifications.length).toEqual(4) // 2 per topic
```

Comment:
Where is the code that creates these notifications? This is very implicit and order dependent. Perhaps you could make this idempotent like so:
```

Reply:
Enabling zeromq is set in bitcoind configuration, and it was already enabled. So that call is not being affected by this branch.

Reply:
I don't understand. This is a new test right? So can we try to make any new tests we add be idempotent?

Reply:
It is a newly added test yes, but is not actually testing anything that has a "before" and "after". It is just detecting if the bitcoind client has the zeromq notifications enabled.